cmd/upgrade/internal/powershell/powershell.go

```diff
@@ -0,0 +1,63 @@
+package powershell
```

Comment:
couldnt this replace /pkg/powershells?

Reply:
I don't think it can without some serious refactoring. `pkg/powershell` executes Powershell commands via a goroutine (so it supports executing a Powershell command in parallel) vs. these are blocking Powershell calls